src/Build/Resources/Strings.resx

```diff
@@ -331,6 +331,18 @@
   </data>
   <data name="TryingExtensionsPath" xml:space="preserve">
     <value>Trying to import {0} using extensions path {1}</value>
+  </data>
+    <data name="OverrideUsingTaskElementCreated" xml:space="preserve">
+    <value>MSB4275: Created an override using task: {0}</value>
+    <comment>{StrBegin="MSB4275: "}</comment>
+  </data>
+  <data name="DuplicateOverrideUsingTaskElement" xml:space="preserve">
+    <value>MSB4276: Attempted to create multiple overrides of the same task: {0}</value>
+    <comment>{StrBegin="MSB4276: "}</comment>
+  </data>
+    <data name="OverriddenUsingTaskElementChosen" xml:space="preserve">
+    <value>MSB4277: Overridden task chosen when calling task: {0}</value>
```

Comment:
Should this have a pointer to the XML location of the override? If it's wrong (which is the only time I can think of to care about this message) that's probably interesting.

Reply:
Now that this message is gone, do you think it'd be useful to log the location when the overridetask is registered? It currently only logs the task name.